src/auth/Login.tsx

```diff
@@ -40,6 +47,18 @@ const Login = () => {
       });
   };
 
+  const getAllOnlineEmailData = async () => {
+    await axios.get('http://kdt.frontend.3rd.programmers.co.kr:5006/users/online-users').then((res) => {
+      const serverData = res.data;
```

Comment:
```suggestion
    await axios.get('http://kdt.frontend.3rd.programmers.co.kr:5006/users/online-users').then(({data}) => {
      const serverData = data;
```
구조분해할당으로 꺼내쓸 수도 있을 것 같아요! 구조분해할당으로 꺼내서 그냥 바로 map으로 돌려도 될 것 같습니다!